developer/decision-records/2024-04-19-tech-repo-split/README.md

```diff
@@ -0,0 +1,31 @@
+# Decision separation build pipelines for Technology Repositories
```

Comment:
The term technology applies also to the EDC core. This might confuse the community. One could assume that everything that isn't technology is documentation, specific to use cases, or industry-specific.

Maybe one of the terms provider, platform, or cloud could be more suitable.